configure.ac

```diff
@@ -430,6 +430,8 @@ MINGW=`echo $host_os | grep mingw32  2>/dev/null`
 AM_CONDITIONAL(MINGW, test ! -z "$MINGW" )
 if test ! -z "$MINGW"; then
 	WSOCKLIB="-lws2_32"
+	LDFLAGS="-no-undefined $LDFLAGS"
+	AC_DEFINE(HAVE_GETTIMEOFDAY)
```

Comment:
Some results from my new tests now using cmake:

. log 1: http://pastebin.com/7WUvXdWr;
. log 2: http://pastebin.com/C3Ld9DKt;
. log 3: http://pastebin.com/9xmrLrt7; (after applying my change in the `rfb.h` file - you can get the change in my patch)
. log 4: http://pastebin.com/tjNCjvr9.

It seems libvncserver really hates Windows environment. :-/